compilecode.c

```diff
@@ -209,6 +156,18 @@ static const char *_compilecode(const char *re, ByteProg *prog)
     return re;
 }
 
+int re1_5_sizecode(const char *re)
+{
+    ByteProg dummyprog = {
```

Comment:
Static will make it necessary to add a statement to assign the bytelen field, since _compilecode() changes it. So there is no benefit in doing so, and I think it is better to leave it auto.